### PR TITLE
Add docker setup

### PR DESCRIPTION
**Pull request by blackthornedk**
This adds rudimentary docker setup.

The container can be built, site and db can be initialized and the container can be started, using make commands.
The makefile is copied more or less from bornhack-website, and the two projects can be run in docker compose, side by side.

### CODE REVIEW

**Comment by blackthornedk** [comment]
I'll add a few things, hold on merging.

**Comment by blackthornedk** [comment]
All done for now, ready for review.

File: src/bma/.env.dist
Comment (0xUnicorn) [inline src/bma/.env.dist:1]: I would prefer to keep the existing `DJANGO_DATABASE_URL`  and use `sed` to change the database address within the docker setup. This way docker stays "self-contained".

PS. I've never tried clicking `Start a review`, I hope no bells&whistles begins.

Comment (blackthornedk) [reply src/bma/.env.dist:1]: This is done entirely different in bornhack-website. The suggestion below originates from @valberg. I have no opinion on whether to use this, the one from bornhack-website, or a third one?

Comment (blackthornedk) [reply src/bma/.env.dist:1]: Branch has been updated.

**Review by 0xUnicorn — APPROVED** [review]
:star: Nice

**Pull request merged**